### PR TITLE
Specify the requirements on the compression function.

### LINKED ISSUE #16: Evaluate potential attack
See message on mailing list from @siyengar for more details:
https://www.ietf.org/mail-archive/web/tls/current/msg25851.html

### PR DESCRIPTION
**Pull request by vasilvv**
Fixes #16

### CODE REVIEW

File: draft-ietf-tls-certificate-compression.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ After decompression, the Certificate message MUST be processed as if it were
 encoded without being compressed.  This way, the parsing and the verification
 have the same security properties as they would have in TLS normally.
 
+In order for certificate compression to function correctly, the underlying
+compression algorithm MUST be determenistic and it MUST output the same data
```
Comment (ekr) [inline draft-ietf-tls-certificate-compression.md]: deterministic

perhaps "no compressed data can map to either two inputs or two outpots"

Comment (vasilvv) [reply draft-ietf-tls-certificate-compression.md]: Fixed the typo.

I'm not sure what "compressed data map to inputs" mean?  I think the requirements on compression functions f/f^{-1} that we want is f^{-1}(f(x)) = x for all x, which I feel like I've stated here.

Comment (ghedo) [reply draft-ietf-tls-certificate-compression.md]: You could add that there shouldn't be compression collisions (which is a term that was floated during the session last week).

**Comment by ghedo** [comment]
Thanks for doing this @vasilvv, I ended up being tied up in a bunch of other stuff.

**Review by ghedo — APPROVED** [review]
lgtm

**Pull request merged**